### PR TITLE
Testing and fixing support for shadow CSS value parsing.

### PR DESCRIPTION
**Pull request by mithro**

### CODE REVIEW

File: web-animations.js [hunk 1]
```diff
@@ -2900,41 +2900,52 @@ var shadowType = {
...
+    }
+
     var result = shadows.map(function(value) {
       if (value === 'none') {
         return shadowType.zero();
       }
-      value = value.replace(/^\s+|\s+$/g, '');
-      var parts = value.split(/\s+/);
-      if (parts.length < 2 || parts.length > 6) {
+      value = value.replace(/^[\s]+|\s+$/g, '');
```
Comment (dstoc) [inline web-animations.js]: existing regex seems fine

File: test/testcases/impl-test-from-css-value.html [hunk 2]
```diff
@@ -30,7 +30,111 @@
...
+ [{hOffset:{px: 10}, vOffset:{px: 15}, blur: {px: 5}}],
+ "no spread");
+
+// Examples from MDN
+assert_shadow_equals(
+ shadowType.fromCssValue("1px 2px 3px black, 0 0 1em blue, 0 0 0.2em blue"),
+ [{hOffset:{px: 1}, vOffset:{px: 2}, blur: {px: 3}, color: [0, 0, 0, 1.0]},
+  {hOffset:{'': 0}, vOffset:{'': 0}, blur: {em: 1}, color: [0, 0, 255, 1.0]},
+  {hOffset:{'': 0}, vOffset:{'': 0}, blur: {em: 0.2}, color: [0, 0, 255, 1.0]}],
+ "MDN Example 1");
```
Comment (dstoc) [inline test/testcases/impl-test-from-css-value.html:109]: this should describe what is being tested, not where the test came from

Comment (dstoc) [inline test/testcases/impl-test-from-css-value.html:109]: also, single quotes...

File: web-animations.js (hunk 1 above)
Comment (mithro) [inline web-animations.js]: Done.

File: test/testcases/impl-test-from-css-value.html (hunk 2 above)
Comment (mithro) [inline test/testcases/impl-test-from-css-value.html:109]: Rest of this file seems to use double quotes. I'll go through and replace with double for now and then do a search/replace in another pull request.

Comment (mithro) [inline test/testcases/impl-test-from-css-value.html:109]: These are a bunch of "real world" tests to make sure that my theoretical tests don't miss something.

**Comment by mithro** [comment]
PTAL.

File: web-animations.js [hunk 3]
```diff
@@ -2900,41 +2900,52 @@ var shadowType = {
   // TODO: This should handle the case where the color comes before the
```
Comment (dstoc) [inline web-animations.js]: looks like this is done

**Comment by dstoc** [comment]
lgtm

**Comment by mithro** [comment]
Waiting for travis to approve, then will merge.

**Pull request merged**